src/v/coproc/tests/router_test.cc

```diff
@@ -165,12 +165,12 @@ FIXTURE_TEST(test_coproc_router_giant_fanin, router_test_fixture) {
 }
 
 FIXTURE_TEST(test_coproc_router_giant_one_to_many, router_test_fixture) {
-    const std::size_t n_copros = 50;
-    const std::size_t n_partitions = 10;
-    const model::topic source_topic("sole_input");
+    const std::size_t n_copros = 25;
+    const std::size_t n_partitions = 5;
+    const model::topic source_topic("input");
     const auto range = boost::irange<std::size_t>(0, n_copros);
     ss::do_for_each(range, [this](std::size_t i) {
-        return add_copro<unique_identity_coprocessor>(i, {{"sole_input", l}});
+        return add_copro<unique_identity_coprocessor>(i, {{"input", l}});
```

Comment:
don't you mean `source_topic` no quotes here ?

Reply:
It can really be named anything, i just changed it so the test logs from this test would appear different from the subsequent tests above this